Service/CallService.php

```diff
@@ -190,9 +195,16 @@ public function call(
 //            $this->entityManager->persist($log);
 //            $this->entityManager->flush();
 
+            // TODO: monolog
+            var_dump($e->getMessage());
```

Comment:
ik snap deze niet

Reply:
If callService gets an error response we want to do something with the error, maybe we shouldn't push this for now, i will comment it out